CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - Drop support for Rails 4.2.
 
+### Breaking Changes
```

Comment:
Dropping Rails 4.2 is also a breaking change, so the header should probably be moved up.

Reply:
The reason why I didn't classify it as a breaking change, is because bundler won't let you have this new version that depends on AR >=5.2 and a locked version of AR 4.2